core/backend/src/test/ecdb/ECSqlQuery.test.ts

```diff
@@ -140,20 +140,23 @@ describe("ECSql Query", () => {
     assert.isTrue(reader.stats.totalTime > 0);
   });
   it("concurrent query use idset", async () => {
-    const reader = imodel1.createQueryReader("SELECT * FROM BisCore.element WHERE InVirtualSet(?, ECInstanceId)", QueryBinder.from([["0x3b", "0x39", "0x3a"]]), { usePrimaryConn: true });
+    const ids: string[] = [];
+    for await (const row of imodel1.query("SELECT ECInstanceId FROM BisCore.Element ORDER BY random() LIMIT 23")) {
```

Comment:
What is benefit of adding randomness to test?

Reply:
If the result of the test is timing-dependent then it is by definition still flaky.
Can you test this functionality without relying on it executing slow/fast enough?

Reply:
No it was for testing to see total time can be push over 1ms. Which i failed and it was still flaky. So i remove the assert that was asserting on time to be > 1ms as there is no way to ensure it will be always > 1 ms.

I have remove the random().

Reply:
You have also disabled the test for the browser because "it expected test to run >1ms". Is that still necessary?
Is this test deterministic or does it still depend on timing?

Reply:
No it does not require to run in > 1ms any more.

Reply:
So you are going to remove `skipIfWeb` now?

Reply:
They are two different test

`core/backend/src/test/ecdb/ECSqlQuery.test.ts`
**concurrent query use idset**
Does not require to run under 1ms

`full-stack-tests/core/src/frontend/standalone/ECSqlQuery.test.ts`
**Restart query** 
Skipped for web because of cancel request reaches backend after query has ran successfully on macos